contribute.md

```diff
@@ -97,48 +97,52 @@ In general, any Ruby version supported by Rails version required by Foreman will
 
 The Rails version is found by opening the [Gemfile](https://github.com/theforeman/foreman/blob/develop/Gemfile) and searching for `gem 'rails'` line. Then find out which Ruby versions are supported for that particular Ruby on Rails version. Another, perhaps faster way, is to visit our [Jenkins CI server](http://ci.theforeman.org/view/Foreman%20pipeline/job/test_develop) to see matrix of supported versions. Then pick any version from that list.
 
-Instructions for Fedora, CentOS 8, CentOS 8 Stream or Red Hat Enterprise Linux 8:
+Instructions for Enterprise Linux (Almalinux, CentOS Stream, Red Hat Enterprise Linux, Rocky Linux) 9:
 
 ```
-dnf module enable ruby:2.7
+# (OPTIONAL) Change Ruby module version before installing
+dnf module enable ruby:3.3
+# (OPTIONAL) Switch Ruby module version if packages are already installed
+dnf module switch-to ruby:3.3
```

Comment:
At this moment Foreman itself doesn't support Ruby 3.3. We currently use the base version in EL9 so you can leave out the DNF modularity

Reply:
Added a note for "future proofing" so that when it will be needed there already is a reminder that Ruby is shipped as a module